src/requests/team.ts

```diff
@@ -263,6 +270,16 @@ async function getBySlug(client: GraphQLClient, slug: string) {
             logo
             favicon
           }
+          team_settings {
```

Comment:
Have a look at all places where `getBySlug()` is used - it may not be the appropriate place to get the settings from. 

If `getBySlug()` is used in a public context (I think it is), the request may fail as the public role cannot (and should not) access team settings.

We may be better off with a more specific and tightly scoped `team.getSettings(teamSlug: string): TeamSettings` method 👍